.github/workflows/setup-gcloud-it.yml

```diff
@@ -16,7 +16,12 @@ jobs:
         operating-system: [ubuntu-latest, windows-latest, macos-latest]
     steps:
     - uses: actions/checkout@v2
-
+    - id: build
```

Comment:
Don't we need to build the setupGcloudSdk here too?

Reply:
ah yes. This was correctly surfaced when I pushed the first commit https://github.com/GoogleCloudPlatform/github-actions/runs/1217429019?check_suite_focus=true but what not triggered for subsequent commits because of push-path. Multi repo should catch these 😅

Reply:
fixed